src/redist/targets/GenerateBundledVersions.targets

```diff
@@ -269,4 +270,13 @@ Copyright (c) .NET Foundation. All rights reserved.
                       Lines="$(BundledBundledCliToolsPropsContent)"
                       Overwrite="true" />
   </Target>
+
+  <Target Name="LayoutRuntimeGraph"
+          DependsOnTargets="GenerateBundledVersionsProps">
+
+    <Copy SourceFiles="$(NuGetPackageRoot)/microsoft.netcore.platforms/$(_NETCorePlatformsPackageVersion)/runtime.json"
+          DestinationFiles="$(SdkOutputDirectory)runtime.json" 
```

Comment:
Let's pick a more specific name for this that indicates that it is not an arbitrary runtime.json.